src/media/css/feed/feed.styl

```diff
@@ -182,6 +183,13 @@
             text-overflow: ellipsis;
             white-space: nowrap;
         }
+        .desc {
+            font-size: 16px;
+            font-weight: 200;
+            line-height: 18px;
+            margin: 5px auto 0;
+            max-width: $mobile-tile-width - $horizontal-tile-padding;
```

Comment:
give it the max-width of mobile-tile-width if we need to, but use padding instead of subtracting from the width